### PR TITLE
background color

### PR DESCRIPTION
**Pull request by dVoorhout**
Sets the VR background color to blue, can't test it with VR right now, but if anything, it doesn't break the app.

### CODE REVIEW

File: ContextDesktop/src/main/java/com/github/migi_1/Context/model/MainEnvironment.java [hunk 1]
```diff
@@ -576,4 +578,13 @@ public BoundingBox getLeftBound() {
     public BoundingBox getRightBound() {
         return boundingBoxWallRight;
     }
+
+    /**
+     * @param app the app to set
+     */
```
Comment (LMiljak) [inline ContextDesktop/src/main/java/com/github/migi_1/Context/model/MainEnvironment.java:610]: Doesn't Checkstyle complain about this Javadoc?
It doesn't contain a description (that has to end with a dot).
Just a parameter description is not enough.

**Comment by LMiljak** [comment]
Also, Merge conflict. I'm closing it for now.

**Comment by dVoorhout** [comment]
No checkstyle doesn't care about that when it comes to getters and setters. Added it just for you though.

File: ContextDesktop/src/test/java/com/github/migi_1/Context/model/TestMainEnvironment.java [hunk 2]
```diff
@@ -146,8 +146,13 @@ public void setUp() throws Exception {
         Mockito.when(wrapper.getServer()).thenReturn(Mockito.mock(Server.class));
         PowerMockito.whenNew(Platform.class).withAnyArguments().thenReturn(platform);
         PowerMockito.whenNew(CarrierAssigner.class).withAnyArguments().thenReturn(carrierAssigner);
+<<<<<<< HEAD
+        PowerMockito.suppress(PowerMockito.method(MainEnvironment.class, "initLights"));
+=======
         PowerMockito.whenNew(EnemySpawner.class).withAnyArguments().thenReturn(enemySpawner);
+>>>>>>> origin/master
         env = PowerMockito.spy(new MainEnvironment(Mockito.mock(CarrierAssigner.class)));
+        //env.setApp(app);
```
Comment (NilsHullegien) [inline ContextDesktop/src/test/java/com/github/migi_1/Context/model/TestMainEnvironment.java]: In the merge with master you left a merge conflict in the code. This is why the travis build fails. Please fix this

**Comment by NilsHullegien** [comment]
Code looks good. Tested it with the Oculus.

**Comment by LMiljak** [comment]
Merging. :+1:

**Pull request merged**